calendar_demo/calendar_demo/calendar_demo.py

```diff
@@ -7,12 +7,18 @@
 
 class Foo(rx.State):
     selected_date: str = ""
+    start_date: str = ""
+    end_date: str = ""
     logs: list[str] = []
 
     def change_handler(self, var):
         self.selected_date = var
         return self.add_log(f"Changed selected date: {var}")
 
+    def change_range_handler(self, var):
+        self.start_date, self.end_date = var
+        return self.add_log(f"Changed selected date range: {self.start_date} {self.end_date}")
```

Comment:
```suggestion
        return self.add_log(f"Changed selected date range: {self.start_date} - {self.end_date}")
```
nit: for easier readability